RNDeviceInfo.podspec

```
@@ -10,7 +10,7 @@ Pod::Spec.new do |s|
 
   s.authors      = package['author']
   s.homepage     = package['repository']['url']
-  s.platform     = :ios, "9.0"
+  s.platforms     = { :ios => "9.0", :visionos => "1.0" }
```

curious - does this add value given that 1 is the first available?
Stated differently, the only available value is already the minimum so this would not restrict anything, whereas the reason to have iOS 9 is so that pod won't even try on ios 8...

If this is needed, then that would imply we also need a minimum for tvOS iPadOS etc etc ?

---

since visionos is an out-of-the-tree platform, I guess it is needed & the same is suggested to be done in the [blog](https://www.oskarkwasniewski.dev/blog/bringing-react-native-libraries-to-apple-vision-pro#migrating-libraries).

the same change was also done in https://github.com/callstack/react-native-slider/pull/560/files

> If this is needed, then that would imply we also need a minimum for tvOS iPadOS etc etc ?

I did some research but couldn't find it.

@okwasniewski - would appreciate it if you could exchange your thoughts here, this can be a good addition to your blog as well.

---

I'm pretty sure we need to have `visionos` and `tvos` here. iPad is essentially iOS so it's not needed here. And for your comment @mikehardy that's the convention Cocoapods took to specify minimal version so even if we have one available we need to add it 😄

```suggestion
  s.platforms     = { :ios => "9.0", :visionos => "1.0", :tvos => "10.0" }
```

---

Very interesting! Thanks for the feedback. As I was typing out iPadOS I knew that one was really just something I was throwing in as it reduces to iOS. But macOS perhaps 🤔  - easy to defer that but I like your suggestion for tvOS, thank you @okwasniewski